scenarios3/14_Chasing_Dragons.cfg

```diff
@@ -749,6 +852,8 @@
                                     message= _ "Something happened."
                                 [/message]
                                 {EARTHQUAKE (    {MODIFY_UNIT side=2 status.petrified no})}
+                                # on hard, consider adding objective here to keep side 2 leader alive.  He shows no interest in self-preservation, so it could be a challenge.
+                                # of course, on hard if you trigger these guys you should be ashamed of yourself.
```

Comment:
Might be worth the little change it takes.

Reply:
ok, made it an option